grails-app/assets/javascripts/streama/templates/admin-show.tpl.htm

```diff
@@ -115,7 +115,7 @@
 <div ng-show="!showLoading">
 
   <button class="btn btn-primary" ng-click="addNewEpisode()">Add New Episode Manually</button>
-  <button class="btn btn-primary" ng-click="fetchAllEpisodesForSeason()" ng-if="!show.manualInput">Fetch all Episode for Season</button>
+  <button class="btn btn-primary" ng-click="fetchEpisodes()" ng-if="!show.manualInput">Fetch Episodes</button>
```

Comment:
did you adjust the existing buttons functionality? How exactly does it work? if the user does not choose a season number, it will fetch all?